### PR TITLE
ath79: Add support for TP-Link Archer C59v2

### PR DESCRIPTION
**Pull request by kicken**
Adds support for TP-Link Archer C59v2 to the ath79 platform.

Based on changes made in PR #2364

### CODE REVIEW

**Comment by adrianschmutzler** [comment]
Hi, I would actually just copy the device specs and flashing instructions from the ar71xx commit message instead of referring to them.

**Comment by adrianschmutzler** [comment]
Note for reviewers:
This will require the same LED migration as proposed in https://patchwork.ozlabs.org/patch/1156538/

However, as long as the referred patch is on hold, I would not require this PR to care about migration.

File: target/linux/ath79/dts/qca9561_tplink_archer-c59-v2.dts [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+};
+
+&leds {
+    usb {
+        label = "tp-link:green:usb";
+        gpios = <&led_gpio 7 GPIO_ACTIVE_LOW>;
+        linux,default-trigger = "usbport";
+        trigger-sources = <&hub_port>;
+    };
+};
```
Comment (adrianschmutzler) [inline target/linux/ath79/dts/qca9561_tplink_archer-c59-v2.dts]: Indent in the block above is wrong (spaces instead of tabs).

**Review by adschm — APPROVED** [review]
Should be ready for merge.
My patch can be used from this source, there is no difference to the version in the earlier separate PR.
@blocktrron

File: target/linux/ath79/dts/qca9561_tplink_archer-c59-v2.dts (hunk 1 above)
Comment (adrianschmutzler) [reply target/linux/ath79/dts/qca9561_tplink_archer-c59-v2.dts]: Done.

**Comment by blocktrron** [comment]
Thanks! Merged to my staging tree.

**Comment by adrianschmutzler** [comment]
In your staging tree it looks like the second patch has author-signed-off mismatch in surname ...

**Comment by blocktrron** [comment]
Yes, I've missed reopening this PR yesterday.

@kicken Can you fix the commit author for your second patch?

**Comment by adrianschmutzler** [comment]
@kicken If your Signed-off is correct, the easiest is:
`git commit --amend --reset-author` 
This will reset the author to your current signed-off.

**Comment by kicken** [comment]
@blocktrron Done.

**Pull request closed without merging**